UI/admin/dishes.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
add a `lang` attribute in the opening `html` tag.   seems like your using html 4 . u need to use html 5 , probably your missing out on some newfeatures

Reply:
Thanks for the feeback, let me work on that